Automattic/newspack-theme
# Add stylesheets, build process and fonts for 'Style D'

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds stylesheets for 'Style D', adds them to the build process, and enqueues the style's font.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`; make sure there are no errors in the build process.
2. Navigate to Customize > Style 3.
3. Confirm that the font "Barlow" is used for the headers and body text.

### Other information:
